ormar/queryset/queryset.py

```diff
@@ -1076,11 +1080,14 @@ async def bulk_update(  # noqa:  CCR001
 
         Bulk operations do not send signals.
 
-        :param objects: list of ormar models
+        :param objects:list of ormar models
         :type objects: List[Model]
         :param columns: list of columns to update
         :type columns: List[str]
         """
+        if not objects:
+            raise ModelListEmptyError("Bulk update objects are empty!")
```

Comment:
This one is kind of a breaking change, but I can agree that raising an explicit exception on an empty list makes sense and shouldn't actually break much.

Reply:
maybe should support

ormar.datetime(onupdate=datetime.now)

Reply:
```
if not objects:
    logger.warning('bulk update objects are empty')
    return
```

And,  log the empty and return here only?